ps_landing_page/models/hr_employee_landing_page.py

```diff
@@ -260,29 +306,42 @@ def action_view_analytic_tree(self):
         self.ensure_one()
         ir_model_data = self.env["ir.model.data"]
         tree_res = ir_model_data.get_object_reference(
-            "ps_timesheet_invoicing", "view_ps_time_line_tree"
+            "ps_landing_page", "view_ps_time_line_landing_page_tree"
```

Comment:
please don't duplicate views for minor adjustments, this makes maintenance more difficult in the long run. My version of this is https://github.com/tosccolors/vertical-professional-services/pull/12, making the amount column invisible based on a context key we set here, then you don't need a new view.

In case you ever really need a different view for adjustments, inherit from the one you'd like to duplicate with mode `primary`, then you can only add changes and don't need to copy the whole thing. A good example for this are the [list views of invoices](https://github.com/OCA/OCB/blob/14.0/addons/account/views/account_move_views.xml#L439-L465), they both inherit from a shared list and change only what is necessary. Much more maintainable this way.

Reply:
@hbrunn Noted!